mopro-core/src/middleware/gpu_explorations/utils/preprocess.rs

```diff
@@ -91,7 +89,7 @@ impl Iterator for FileInputIterator {
             Err(_) => None,
         }?;
 
-        let scalars = Vec::<Scalar>::deserialize(&self.scalars_file);
+        let scalars = Vec::<Scalar>::deserialize_compressed(&self.scalars_file);
```

Comment:
can also do `deserialize_compressed_unchecked`?
suppose to be faster

Reply:
Thanks, I'll give it a try